README.md

```diff
@@ -327,6 +357,13 @@ Improve your projects using these awesome illustrations. See a sample <a href="h
             <sub style="font-size:14px"><b>David</b></sub>
         </a>
     </td>
+  <td align="center" style="word-wrap: break-word; width: 150.0; height: 150.0">
+        <a href=https://github.com/jpacsai>
+            <img src="https://avatars.githubusercontent.com/u/34191321?s=400&u=7e9e287240029b1d3e44db5cc96940c83546f82d&v=4" width="100;"  style="border-radius:50%;align-items:center;justify-content:center;overflow:hidden;" alt=David/>
```

Comment:
- change url src to `https://avatars.githubusercontent.com/u/34191321?v=4`

- change the alt text

- I would highly suggest not doing the maintainer's work since correcting you seems like extra effort for me.

Reply:
I have added this in contributors, so you can remove this.

Reply:
apologies, I thought the contributors need to to their own avatars if they wish to be included. sorry